mantl/mantl
# add certificate rotation playbook

- [x] Installs cleanly on a fresh build of most recent master branch
- [x] Upgrades cleanly from the most recent release
- [x] Updates documentation relevant to the changes

fixes #1605 

to test: 
1. create a normal deployment
2. log onto a box and save the output of `openssl x509 -in /etc/pki/mantl -noout -text`
3. run playbooks/rotate-certs.yml
4. confirm that the cluster is still healthy
5. log into that same box and confirm that the same command gives different (yet still appropriate) output

next steps:
1. Use TLS auth 
2. Optionally generate new root CA in Vault 
3. Optionally generate intermediate CA in Vault
